cmd/server/server.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func NewServer() *http.Server {
+	r := http.DefaultServeMux
+	// refactor this, maybe another package for handlers ? 
+	r.HandleFunc("/health", func(rw http.ResponseWriter, r *http.Request) {
+		rw.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(rw).Encode(struct{ Ok bool }{Ok: true})
+	})
+	server := http.Server{
+		Addr:    "localhost:3000",
+		Handler: r,
```

Comment:
**Suggestion:**
it would be cool if we can make this dynamic by using the `Configs` struct.

Reply:
Should we use a lib or only a struct?